modules/decrypt-node/test/parse_header_stream.test.ts

```diff
@@ -8,22 +8,41 @@ import chaiAsPromised from 'chai-as-promised'
 import * as util from 'util'
 import * as stream from 'stream'
 const pipeline = util.promisify(stream.pipeline)
-import { ParseHeaderStream } from '../src/parse_header_stream'
+import {
+  ParseHeaderStream,
+  ParseHeaderStreamOptions,
+} from '../src/parse_header_stream'
 import {
   NodeDefaultCryptographicMaterialsManager,
+  NodeAlgorithmSuite,
   needs,
 } from '@aws-crypto/material-management-node'
+import { deserializeFactory } from '@aws-crypto/serialize'
 import * as fixtures from './fixtures'
 chai.use(chaiAsPromised)
 const { expect } = chai
 
-describe('ParseHeaderStream', () => {
+const toUtf8 = (input: Uint8Array) =>
+  Buffer.from(input.buffer, input.byteOffset, input.byteLength).toString('utf8')
+const { deserializeMessageHeader } = deserializeFactory(
+  toUtf8,
+  NodeAlgorithmSuite
+)
+
+describe.only('ParseHeaderStream', () => {
   it('Postcondition: A completed header MUST have been processed.', async () => {
-    const completeHeaderLength = 73
     const data = Buffer.from(
       fixtures.base64CiphertextAlgAes256GcmIv12Tag16HkdfWith4Frames(),
       'base64'
     )
+
+    const headerInfo = deserializeMessageHeader(data)
+    needs(headerInfo, 'No header, test impossible')
+    const completeHeaderLength =
+      headerInfo.rawHeader.byteLength +
+      headerInfo.algorithmSuite.ivLength +
+      headerInfo.algorithmSuite.tagLength / 8
```

Comment:
It would be good to explicitly compute `completeHeaderLength` this way in the next text fixture as well (`it('Precondition: If maxHeaderSize was set...')`)

Reply:
reasonable, I should fix this elsewhere as well.
But I'll do that in a different PR